descarteslabs/services/metadata.py

```diff
@@ -493,7 +501,7 @@ def keys(self, products=None, const_id=None, sat_id=None, date='acquired', place
                              end_time=end_time, cloud_fraction=cloud_fraction,
                              cloud_fraction_0=cloud_fraction_0, fill_fraction=fill_fraction,
                              params=params, limit=limit, offset=offset, fields=["key"], dltile=dltile,
-                             sort_field=sort_field, sort_order=sort_order)
+                             sort_field=sort_field, sort_order=sort_order, randomize=None)
```

Comment:
This should be `randomize=randomize` right? do you not want to allow randomized keys?